.github/workflows/test_migrations.yaml

```diff
@@ -0,0 +1,246 @@
+name: Test Skyportal Migrations
+
+on:
+  # Run only if changes in Alembic versions
+  pull_request:
+    paths:
+      - "skyportal/models.py"
```

Comment:
Might be worth tracking changes to the pinned baselayer/app/models.py, too

Reply:
I imagine it will be the same case here as I indicated in https://github.com/skyportal/skyportal/pull/1726#discussion_r585081969

Reply:
@acrellin Update here as well to just run on all changes to "baselayer"

Reply:
Great, I was worried about unnoticed baselayer model changes going in with a missing or incomplete migration script.